packages/fxa-payments-server/package.json

```diff
@@ -126,7 +126,7 @@
     "dayjs": "^1.9.4",
     "eslint-plugin-jest": "^24.5.2",
     "express": "^4.17.1",
-    "fxa-content-server": "workspace:*",
```

Comment:
this would create a cyclic dependency. content-server functionally depends on payments-server, currently all the functional tests are there, while payments-server only depends on some css which it accesses directly, i.e. `../../.......`. Both the css dep and the functional tests all being under content-server are issues for another day